chart/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1 @@
+# lol no
```

Comment:
I know we're not going to add tests, so we may as well delete this file.

Reply:
fair